src/google/protobuf/compiler/csharp/csharp_source_generator_base.cc

```diff
@@ -56,6 +56,9 @@ SourceGeneratorBase::~SourceGeneratorBase() {
 
 void SourceGeneratorBase::WriteGeneratedCodeAttributes(io::Printer* printer) {
   printer->Print("[global::System.Diagnostics.DebuggerNonUserCodeAttribute]\n");
+  // We don't provide protoc info in the [GeneratedCode] attribute the because the
+  // pre-generated code in this repository would then have junk info in it.
```

Comment:
I don't really follow what you mean by "junk info". It might be worth clarifying that the second parameter would be the version information, if we wanted to include that... but I agree that it's not worth including it (as it would create a lot of churn every time the version is updated.)